slices/openjdk-8-jre-headless.yaml

```diff
@@ -0,0 +1,118 @@
+package: openjdk-8-jre-headless
+
+slices:
+  bins:
+    essential:
+      - ca-certificates-java_data
+      - openjdk-8-jre-headless_libs
+      - openjdk-8-jre-headless_policy
+      - openjdk-8-jre-headless_properties
+      - openjdk-8-jre-headless_certs
+      - openjdk-8-jre-headless_data
+      - openjdk-8-jre-headless_jars
+      - openjdk-8-jre-headless_config
+      - openjdk-8-jre-headless_profiles
+    contents:
+      /usr/lib/jvm/java-8-openjdk-*/jre/bin/java:
+
+  certs:
+    contents:
+      /etc/java-8-openjdk/security/blacklisted.certs:
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/security/blacklisted.certs:
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/security/cacerts:
+
+  config:
+    contents:
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/*/jvm.cfg-default:
+
+  data:
+    contents:
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/currency.data:
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/tzdb.dat:
+
+  policy:
+    contents:
+      /etc/java-8-openjdk/security/java.policy:
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/security/policy/limited/US_export_policy.jar:
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/security/policy/limited/local_policy.jar:
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/security/policy/unlimited/US_export_policy.jar:
+      /usr/lib/jvm/java-8-openjdk-*/jre/lib/security/policy/unlimited/local_policy.jar:
+
+  jars:
```

Comment:
I'm not sure if it makes sense to separate libs and jars. And if you want to keep them separate, then jars should depend on libs, because JDK classes load the native libraries.

Reply:
Good point. I'll add the dependency.

Reply:
done